test/unit/org/apache/cassandra/cql3/validation/operations/SelectTest.java

```diff
@@ -1913,7 +2009,7 @@ public void testAllowFilteringOnPartitionKeyWithoutIndicesWithCollections() thro
         execute("INSERT INTO %s (a, b, c, d, e) VALUES (1, 4, [1, 2], {2, 4}, {1: 2})");
         execute("INSERT INTO %s (a, b, c, d, e) VALUES (2, 3, [3, 6], {6, 12}, {3: 6})");
 
-        beforeAndAfterFlush(() -> {
+        //beforeAndAfterFlush(() -> {
```

Comment:
Why is this commented? It seems to work fine

Reply:
Probably commented for debugging and forgot it, thanks